src/components/SearchResults.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "../styles/searchresults.css";
+import PropTypes from "prop-types"
+
+const SearchResults = ({ results }) => {
+
+    if (!results.length) {
+        return <p>No results</p>
+    } else {
+        return (
+            <div className="search-results">
+                <h1>Image Results:</h1>
+                {
+                    results.map((image, index) => (
+                        <img
+                            key={index}
+                            className="search-results__images"
+                            src={image}
+                            alt="search-results__image"
```

Comment:
For a future improvement, it would be good to include some meaningful text in the alt attribute, there definitely enough data coming from the API to provide it. The point of an `alt` tag is to give screen readers a description of the image for those that aren't able to see it